packages/docs/site/docs/11-architecture/18-host-your-own-playground.md

```diff
@@ -25,9 +23,35 @@ const client = await startPlaygroundWeb({
 });
 ```
 
-## Build static assets
+## Static assets
+
+There are several ways to get the static assets necessary to host the Playground.
+
+In order of convenience and ease:
+
+-   Download pre-built package
+-   Build from fork using GitHub Action
```

Comment:
suggestion: "Fork the repository and build via the GitHub Action job"

Reply:
Sure, that does sounds better ("fork and build" rather than "build from fork").